error.go

```diff
@@ -82,6 +89,7 @@ const (
 	ErrorCodeParameterMissing                       ErrorCode = "parameter_missing"
 	ErrorCodeParameterUnknown                       ErrorCode = "parameter_unknown"
 	ErrorCodeParametersExclusive                    ErrorCode = "parameters_exclusive"
+	ErrorCodePaymentIntentActionRequired            ErrorCode = "payment_intent_action_required"
 	ErrorCodePaymentIntentAuthenticationFailure     ErrorCode = "payment_intent_authentication_failure"
 	ErrorCodePaymentIntentIncompatiblePaymentMethod ErrorCode = "payment_intent_incompatible_payment_method"
 	ErrorCodePaymentIntentInvalidParameter          ErrorCode = "payment_intent_invalid_parameter"
```

Comment:
Should we add `payment_method_microdeposit_verification_amounts_mismatch` and `payment_method_microdeposit_verification_attempts_exceeded` here?

Reply:
No, I mentioned both in the PR descriptions, those should not be public

Reply:
ha! you mentioned these two in the comment so disregard this.